packages/kbn-es/src/cluster.js

```diff
@@ -282,7 +299,14 @@ exports.Cluster = class Cluster {
 
     // once the http port is available setup the native realm
     this._nativeRealmSetup = httpPort.then(async port => {
-      const nativeRealm = new NativeRealm(options.password, port, this._log);
+      const caCert = await this._caCertPromise;
+      const nativeRealm = new NativeRealm({
+        port,
+        caCert,
+        log: this._log,
+        elasticPassword: options.password,
+        protocol: this._ssl ? 'https' : 'http',
```

Comment:
Protocol, typically, includes the `:`, but I'd be open to this being a `ssl` setting instead.